website/content/commands/operator/area.mdx

```diff
@@ -170,6 +156,20 @@ to join the given address.
 
 The return code will indicate success or failure.
 
+#### Command Options
+
+- `-id=<value>` - Looks up the area to operate on by its ID. This can be given
+  instead of a peer datacenter.
+
+- `-peer-datacenter=<value>` - Looks up the area to operate on by its peer
+  datacenter. This can be given instead of an ID.
```

Comment:
```suggestion
- `-peer-datacenter=<value>` - Specifies the peer datacenter of a network area to operate on. Use this option as an alternative to `-id`.
```

Reply:
Thanks for this comment, I just realized that the term "peer datacenter" here needs to be disambiguated from "peer datacenter" in the context of cluster peering (which is related, but not the same / incompatible with network areas). I'll see that we address that in a subsequent PR.